net.sourceforge.GTKWave.yml

```diff
@@ -0,0 +1,42 @@
+app-id: net.sourceforge.GTKWave
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: gtkwave
+rename-icon: gtkwave
+rename-desktop-file: gtkwave.desktop
+rename-appdata-file: gtkwave.appdata.xml
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --socket=fallback-x11
```

Comment:
Have you tested this on a wayland session, since you build Tk.

Reply:
Yes. I'm able to open the app and use it normally without it crashing.

Reply:
so what is Tk for?

Reply:
I wasn't able to build the application without it.